### PR TITLE
Center document widget and top bar title together

### PR DESCRIPTION
**Pull request by SubordinalBlue**
**Update**: as of 95c5527, PR is now "complete", but of course open for requested changes.

Just two commits so far, with commentary in each: first to clean up a tad; second to get centering math in agreement for both the DocumentWidget and the top bar title.

There is an issue, existing before this patch, of duplication of the logic for positioning the Hermes' DocumentWidget; addressing that remains for a further commit.

I wanted to open this PR now both for feedback on the centering changes and, to be open to advice on how to address the duplicated code.

Will add some test images in a further comment.

### CODE REVIEW

**Comment by SubordinalBlue** [comment]
## Test images.
All are details from F2 screenshots inside a graphics editor to over-lay guide lines, etc.

Full width view, top portion of a quest screen with sidebar. The editor-added green boxes have equal pixel width (UI=4).
image

Tighter details, same layout, at different UI scales `5` to `1`:
 5 | 4 | 3 | 2 | 1       
---|---|---|---|---
image|image|image|image|image

Details for the no-side-bar case look the same, with the guide line being at 960px on a 1920px-wide MC screen.

**Comment by SubordinalBlue** [comment]
Added two further commits:
- eb36683 for edits that shouldn't have "escaped" the previous commit, my bad.
- 95c5527 for consolidating duplicated content area calculations.

Further details in those commits' messages.

File: common/src/main/java/earth/terrarium/heracles/client/screens/AbstractQuestScreen.java [hunk 1]
```diff
@@ -32,6 +32,12 @@ public abstract class AbstractQuestScreen<T> extends BaseCursorScreen {
 
     protected final T content;
 
+    protected static final float SIDE_BAR_PORTION = 0.25f;
+    protected static int SIDE_BAR_WIDTH;
```
Comment (ThatGravyBoat) [inline common/src/main/java/earth/terrarium/heracles/client/screens/AbstractQuestScreen.java]: should be lowercase since its not a constant

**Review by ThatGravyBoat — CHANGES_REQUESTED** [review]

File: common/src/main/java/earth/terrarium/heracles/client/screens/AbstractQuestScreen.java (hunk 1 above)
Comment (SubordinalBlue) [reply common/src/main/java/earth/terrarium/heracles/client/screens/AbstractQuestScreen.java]: Good catch. Fixed in a64366c

**Review by ThatGravyBoat — APPROVED** [review]

**Comment by ThatGravyBoat** [comment]
After conflict is fixed it can be merged 👍

**Comment by SubordinalBlue** [comment]
Fixed.

**Pull request merged**